apps/geo/models.py

```diff
@@ -19,6 +21,7 @@ class Region(UserResource):
     code = models.CharField(max_length=10)
     title = models.CharField(max_length=255)
     public = models.BooleanField(default=True)
+    published = models.BooleanField(default=False)
```

Comment:
Additional work to do:
- Let's use `is_published`.
- `is_published` should be true for all current regions (we need to make sure that current regions are published)
- For new regions, `is_published` should be False by default and can be set by hitting an endpoint.